src/internal/packager2/create.go

```diff
@@ -64,6 +67,11 @@ func Create(ctx context.Context, packagePath string, opt CreateOptions) error {
 
 	if opt.SBOMOut != "" {
```

Comment:
I believe I traced the logic between the earlier `CreatePackage()` and this point - and am curious if there is any value in augmenting this conditional with `&& !opt.SkipSBOM` ?

Sounds like a niche edge-case. But potentially conflicting arguments (from either CLI or SDK use) could potentially result in the SBOM being skipped but `SBOMOut` not being empty and therefor this error printing? 

No hard opinions on this from me personally - but stood-out as a likely scenario for this error to be hit (especially if the SDK use had a static path mentioned etc).

Reply:
This is a great point, it brings into question where the validation should take place, but I think if a user had both the `--skip-sbom` flag and the `sbom-out` flags set they should error. I'm not sure if I should add that validation to the CLI level, the function level, or both. Leaning towards add it to the `Create` function